packages/KPagination/KPagination.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+  <nav aria-label="Pagination Navigation">
+    <div
+      v-if="totalCount > currentPageSize"
+      class="card-pagination-bar">
+      <span class="pagination-text">
+        <span class="pagination-text-pages">{{ pagesString }}</span>
+        {{ pageCountString }}
+      </span>
+      <ul class="pagination-button-container">
+        <li
+          :class="{ disabled: backDisabled }"
+          class="pagination-button square"
+          aria-label="Go to the previous page"
+          @click="pageBack"
+        ><a>
+          <KIcon
+            :color="backDisabled ? 'var(--grey-500)' : 'var(--blue-400)'"
+            icon="arrowLeft"
+            width="16px"
+            heigth="16px"
+            view-box="0 0 16 14"
+          />
+        </a></li>
+        <li
+          v-if="firstDetached"
+          class="pagination-button"
+          aria-label="Go to the first page"
+          @click="changePage(1)"
+        ><a>
```

Comment:
anchor needs an href. If it's not to a page, it should be `#`

Reply:
Fixed